bitnami/metallb/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: {{ .Values.networkPolicy.apiVersion | default "networking.k8s.io/v1" }}
```

Comment:
Could you add this to the `_helpers.tpl` file as we do in Redis: https://github.com/bitnami/charts/blob/74b079cce2bc3efbd2c97cb473e0e5480ed9bc5e/bitnami/redis/templates/_helpers.tpl#L34?